src/fluree/db/peer/http_api.clj

```diff
@@ -863,9 +863,9 @@
         ;; final 404 fallback
         (constantly not-found))
 
+      (->> (wrap-errors (:debug-mode? system)))
```

Comment:
Just for my education - what does moving this up the chain change?

Reply:
It seemed more appropriate because the first middleware wrapper's handler will invoke all of the others when it calls `(handler request)` whereas it will have already returned if it's lower down in the list. So this ordering should catch errors even if they are thrown in middleware when it does `(try (handler request) (catch ...))`.